content/applications/sales/crm/acquire_leads/convert.rst

```diff
@@ -2,66 +2,121 @@
     :align: center
+    :alt: Create opportunity pop-up.
+
+Under the :guilabel:`Customer` heading, choose from the following options:
+
+- :guilabel:`Create a new customer`: Choose this option to use the information in the lead to create
+  a new customer.
+- :guilabel:`Link to an existing customer`: Choose this option, then select a customer from the
+  resulting drop-down menu, to link this opportunity to the existing customer record.
+- :guilabel:`Do not link to a customer`: Choose this option to convert the lead, but not link it to
+  a new or existing customer.
+
+Lastly, when all configurations are complete, click :guilabel:`Create Opportunity`.
+
+.. _sales/crm/acquire_leads/merge-leads:
 
-Would you find a duplicate yourself, ...you can also merge opportunities
-or leads even if the system doesn't propose it.
+Merge leads and opportunities
+=============================
 
-Here’s how, from the list view. Select the opportunities or leads you
-want to merge and the action button will appear. Then, you can select merge.
+Odoo automatically detects similar leads and opportunities by comparing the email addresses of the
+contact names. If a similar lead/opportunity is found, a :guilabel:`Similar Lead` smart button is
+added to the top of the lead/opportunity record.
 
-.. image:: convert/convert_05.png
+.. image:: convert/convert-leads-similar-leads.png
     :align: center
+    :alt: Similar leads smart button.
+
+To compare the details of the similar leads/opportunities, click the :guilabel:`Similar Leads`
+button. This opens a kanban view with only the similar leads/opportunities. Click into each card
+to view the details for each lead/opportunity, and confirm if they should be merged.
+
+.. important::
+   When merging, Odoo gives priority to whichever lead/opportunity was created in the system first,
+   merging the information into the first created lead/opportunity. However, if a lead and an
+   opportunity are being merged, the resulting record is referred to as an opportunity, regardless
+   of which record was created first.
+
+Once it's confirmed the leads/opportunities should be merged, return to the kanban view using
+breadcrumbs, or by clicking the smart button. Click the :guilabel:`☰ (three vertical lines)` icon
+to change to list view.
+
+Check the box on the left of the page for the leads/opportunities to be merged. Then, click the
+:guilabel:`Action ⚙️ (gear)` icon at the top of the page, to reveal a drop-down menu. From that
+drop-down menu, select the :guilabel:`Merge` option to merge the selected opportunities (or leads).
+
+When :guilabel:`Merge` is selected from the :guilabel:`Action ⚙️` drop-down menu, a
+:guilabel:`Merge` pop-up window appears. In that pop-up window, decide to :guilabel:`Assign
+opportunities to` a :guilabel:`Salesperson` and/or :guilabel:`Sales Team`.
+
+Below those fields, the :guilabel:`Select Leads/Opportunities` to merge, along with their related
+information, can be found.
```

Comment:
I think this needs to be rephrased. The guilabel doesn't fit into this sentence grammatically.